src/TodoList.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+class TodoList extends React.Component {
```

Comment:
you can use Function Component here if you dont use `state`

Reply:
yes you really can use functional component